pymatgen/core/bond_lengths.json

```diff
@@ -161,6 +161,15 @@
     ],
     "bond_order": 1.0
   },
+  {
+    "ref": "",
```

Comment:
@kwaters4 Could you add the reference here? Reference is at the footer of the page you linked. Thanks!

Reply:
Looking for it right now,one sec, looking for the right one. Added the wrong one in the last commit.